### PR TITLE
Removes chemistry dead code

### PR DESCRIPTION
**Pull request by help-maint**
This code is no longer used anywhere and there are quite a few empty hooks that never actually defined.
This PR has no effect.

### CODE REVIEW

**Comment by Kurfursten** [comment]
Some of these hooks could be useful to have in the future though, or inspire coders to use them

**Comment by help-maint** [comment]
Want me to comment them out instead? Currently they are completely unimplemented

**Comment by Kurfursten** [comment]
Yeah, I like the idea of keeping them there for inspiration.

File: code/modules/reagents/Chemistry-Holder.dm [hunk 1]
```diff
@@ -371,16 +371,6 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 			R.on_mob_life(M)
 	update_total()
 
-/datum/reagents/proc/conditional_update_move(var/atom/A, var/Running = 0)
```
Comment (help-maint) [inline code/modules/reagents/Chemistry-Holder.dm:374]: I have no idea what this even is

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -192,29 +192,16 @@
...
-/datum/reagent/proc/on_update(var/atom/A)
-	return
-
 /datum/reagent/proc/on_removal(var/data)
 	return 1
 
 /datum/reagent/proc/on_overdose(var/mob/living/M)
 	M.adjustToxLoss(1)
 
-/datum/reagent/proc/OnTransfer()
```
Comment (help-maint) [inline code/modules/reagents/Chemistry-Reagents.dm:215]: Given a better name/implemented in the midazoline PR

File: code/modules/reagents/Chemistry-Holder.dm (hunk 1 above)
Comment (Kurfursten) [reply code/modules/reagents/Chemistry-Holder.dm:374]: It's unused

**Pull request merged**